### ISSUE TITLE
Dircolors theme?

### ISSUE THREAD

**Issue by scottwillmoore** [arcticicestudio/nord#10]
It would be awesome if we could create a dircolors theme. We could structure the theme from the [solarized dircolors repository](https://github.com/seebi/dircolors-solarized).

Currently, folders that contain the permissions `777` are difficult to read, as seen below.

> Unfortuantely many of my files contain this permission, due to being mounted with a NTFS-3G UserMapping file, in order to preserve windows compatibility.

![image](https://cloud.githubusercontent.com/assets/9512557/21639910/c545733c-d2c6-11e6-8a6f-9a732a9da97a.png)

I haven't discovered any other major issues, however I'm sure there are minor tweaks which can improve the dircolors experience.

You thoughts would be greatly appreciated!

NOTE: I haven't thoroughly researched dircolors yet, however there may be a simpler fix for folders containing the `777` permission...

**Comment by scottwillmoore** [comment]
After some investigation, the following information could help.

````sh
# describes executable which generates LS_COLORS variable.
man dircolors
# describes the configuration file which dircolors uses.
man dir_colors
````

[This Stack Exchange page](http://unix.stackexchange.com/questions/241726/fix-ls-colors-for-directories-with-777-permission) documents the issue that I have encountered above, and describes how to fix it. As noted, when editing the `.dircolors` file, Vim will display the colours in the source code. Using `vim nocursorline`, stops Vim from hiding the colours on your current line.

Whether we want to attempt a full customised configuration, or just to document how to fix this issue. Scrolling through the file shows that `777` folders are pretty much the only unreadable entry. The rest of the colours appear sane.

**Comment by arcticicestudio** [comment]
Sounds like a nice port project idea 😄 
I've configured my local dircolors to match the Nord theme, but haven't thought about the fact that users would also like to have matching colors.
I'll create a repository in the next days. It shouldn't take too long to create the main `.dircolors` file.
And thanks for your research investigations 😉

**Comment by scottwillmoore** [comment]
Awesome, thank you so much!

**Comment by arcticicestudio** [comment]
The [Nord dircolors](https://github.com/arcticicestudio/nord-dircolors) port project is now available and the the initial release version [0.1.0](https://github.com/arcticicestudio/nord-dircolors/releases/tag/v0.1.0) has been shipped 🚢

**Comment by scottwillmoore** [comment]
Amazing! Again, thank you so much. :+1:
